CODE_OF_CONDUCT.md

```diff
@@ -1,100 +1,103 @@
 # CMPSC 100: Course Contract
 
-## A word from the Mayor
+## A word from the Mayor's Office
 
-We're building a world. That task shouldn't be taken on as a solitary or light burden. In many ways, the product of such an endeavor always means to be imperfect; we learn more as we _do_, and that process is rarely (i.e. never) linear or "clean." We benefit from being able to be a bit messy. You may intuit from this statement that we mean to engage with _process_ much more than _product_, though the latter is necessary. Following the various processes of `term-world` closely should yield a successful product; however, production of output is only the _basic_ expectation of the work we'll complete together.
+We're building a world. 
 
-That's why this community is evaluated on a _contract basis_, the terms of which are spelled out here. This document specifies your responsibilities and how that work will be rewarded. We're able to substantially alter the terms of this deal (see [the "Rules Rule"](#the-rules-rule) below), though we don't do so without participation in course governance.
+That task shouldn't be taken on as a solitary or easy burden. In many ways, world-bulding is always imperfect; we learn more as we _do_, and that process is rarely (i.e. never) linear or "clean." We benefit from being able to be messy and make many mistakes. You may gather from this statement our approach cares about _process_ much more than _product_, though the latter is necessary. Following the various processes of `term-world` closely should yield a successful product. That written, production is a _basic_ expectation of the work we'll complete together. Once the process is over, we need to have a result.
 
-## Contract stipulations
-
-### The Basic Grade
-
-(To qualify for this tier, all basic assignment objectives must be completely met.)
-
-* All citizens who meet the covenants of this contract will earn an individual letter grade of `C` in this course
+However, there are manifold tasks involved that are often under- or _not_ counted in traditional grading. That's why community members are evaluated on a _contract basis_, the terms of which are spelled out here. This document specifies your responsibilities and how that work will be rewarded. 
```

Comment:
The "under- or _not_ counted" bit is a bit difficult to read, imho.

Reply:
We'll go with _undercounted_.